### DISCUSSION TITLE
code-server 3.9.4 release update to 1.55/1.56 vscode - addresses bugs.

### DISCUSSION THREAD

**Discussion by MxD-js** [coder/code-server#3323] [Q&A] [answered]
Hi, 

When will code-server sport the 1.55 or 1.56 version of vscode. Microsoft addressed few bugs in Java Debugger extension but needs minimum 1.55 before I can apply the extension update. 

This is the error that comes up when running debugger. 
![image](https://user-images.githubusercontent.com/60336163/117545873-7ffec100-afdc-11eb-851a-5a6508509b31.png)

`'void org.eclipse.jdi.internal.connect.ConnectorImpl$StringArgumentImpl.<init>(org.eclipse.jdi.internal.connect.ConnectorImpl, java.lang.String, java.lang.String, java.lang.String, boolean)'`

Stack overflow has answer: https://stackoverflow.com/questions/67327031/vscode-java-debugger-throwing-unknown-error

but requires version 1.55 to apply updated extension

I tried to manually update the extension in code-server but got this. 
![image](https://user-images.githubusercontent.com/60336163/117545957-eb489300-afdc-11eb-8eff-98a8bf92d735.png)

Workaround: 
Also related: https://github.com/microsoft/vscode-java-debug/issues/995
Downgrading to 0.77 vscode-java, does in fact fix this error in code-server. 

Thanks,

**Comment by jsjoeio** [comment, marked as answer]
Hi! Thanks for asking. We've upgraded to 1.56 (see https://github.com/cdr/code-server/pull/3269) and this will be included in the next release (see [milestone](https://github.com/cdr/code-server/milestone/21)). Keep an eye out for it this week.

**Reply by MxD-js** [reply]
Awesomeness all around, Updated to 3.10.0 and confirmed issue resolved.
